const connectDB = async () => {
    try {
        
    } catch(err){
        console.log(err);
    }
}